recipes/_master_package.rb

```diff
@@ -65,7 +65,8 @@
 
 # Create/fix permissions on supplemental directories
 %w(cache lib run).each do |folder|
-  directory "/var/#{folder}/jenkins" do
+  directory "fix_permissions_for_/var/#{folder}/jenkins" do
```

Comment:
You don't need the underscores here. The resource block's name can have spaces in it.

Reply:
:+1: